File: Daily Challanges/2026-05-May/2026-05-28.js
function fizzBuzzCount(start, end) {
  const counts = {
    fizz: 0,
    buzz: 0,
  };
  for (let num = start; num <= end; num++) {
    if (num % 3 === 0) counts.fizz += 1;
    if (num % 5 === 0) counts.buzz += 1;
  }
  return counts;
}

console.log(fizzBuzzCount(1, 11));

File: Daily Challanges/2026-05-May/2026-05-28.py
def fizz_buzz_count(start, end):
    counts = {
        "fizz": 0, 
        "buzz": 0
    }
    for num in range(start, end+1):
        if num%3==0 :
            counts["fizz"] += 1
        if num%5==0 :
            counts["buzz"] += 1
    return counts

print(fizz_buzz_count(1, 11))